test/v2/controllers/SubmitEndOfPeriodStatementControllerSpec.scala

```diff
@@ -16,39 +16,33 @@
 
 package v2.controllers
 
-import play.api.libs.json.Json
-import play.api.mvc.{ AnyContentAsJson, Result }
-import uk.gov.hmrc.http.HeaderCarrier
+import play.api.mvc.{AnyContentAsJson, Result}
 import v2.data.SubmitEndOfPeriodStatementData._
 import v2.mocks.MockIdGenerator
 import v2.mocks.requestParsers.MockSubmitEndOfPeriodStatementParser
 import v2.mocks.services._
 import v2.models.domain.Nino
 import v2.models.errors._
 import v2.models.outcomes.ResponseWrapper
-import v2.models.request.{ SubmitEndOfPeriodStatementRawData, SubmitEndOfPeriodStatementRequest }
+import v2.models.request.{SubmitEndOfPeriodStatementRawData, SubmitEndOfPeriodStatementRequest}
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 class SubmitEndOfPeriodStatementControllerSpec
     extends ControllerBaseSpec
+    with ControllerTestRunner
     with MockEnrolmentsAuthService
```

Comment:
I don't think we need to extend the `MockEnrolmentsAuthService`, `MockMtdIdLookupService` or `MockIdGenerator` traits here